cirrus/server/cirrus/sdk.py

```diff
@@ -2,14 +2,47 @@
 import logging
 from typing import Any, Dict, List
 
-from cirrus_sdk import CirrusClient, NimbusError  # type: ignore
+from cirrus_sdk import (  # type: ignore
+    CirrusClient,
+    EnrollmentStatusExtraDef,
+    MetricsHandler,
+    NimbusError,
+)
 
 logger = logging.getLogger(__name__)
 
 
+class CirrusMetricsHandler(MetricsHandler):
+    def __init__(self, metrics: Any, pings: Any):
+        self.metrics = metrics
+        self.pings = pings
+
+    def record_enrollment_statuses(
+        self, enrollment_status_extras: list[EnrollmentStatusExtraDef]
+    ):
+        for enrollment_status_extra in enrollment_status_extras:
+            self.metrics.cirrus_events.enrollment_status.record(
+                self.metrics.cirrus_events.EnrollmentStatusExtra(
```

Comment:
It's kinda weird that the type is exposed as a member instead of a top level type. Is this for test stubbing?

Reply:
Nope, that's just kinda how Glean is used in Kotlin/Swift/Python as far as I'm aware

Reply:
Here's an example in iOS https://github.com/mozilla-mobile/firefox-ios/blob/8967b3f84d9a35f79c9b0b4cc1381a5926df4b4f/Client/Telemetry/SponsoredTileTelemetry.swift#L39C1-L45

Reply:
wack.